README.md

```diff
@@ -166,7 +166,7 @@ You can visite the online version at : <a href="https://grandi0z.github.io/#">gr
 👤 **Author1**
 
 - GitHub: [@Grandi0z](https://github.com/Grandi0z)
-- Twitter: [@wa_inje](https://twitter.com/wa_inje)
+- Twitter: [@wa_inje](https://twitter.com/Joseph_LSK)
```

Comment:
- [ ]  Please check the online version of your portfolio, and make sure that the page works fine. ✔️